### ISSUE TITLE
xpytorch + paddlepaddle-xpu同时import会出现兼容性错误，且为paddle报错

### ISSUE THREAD

**Issue by sungenglab** [PaddlePaddle/Paddle#72088]
### bug描述 Describe the Bug

'''
import numpy as np

import paddle

import torch

# Set the seed for reproducibility
np.random.seed(42)

# Generate random input using NumPy
shape = (3, 3)
input_numpy = np.random.randn(*shape)

# Convert NumPy array to PyTorch tensor
torch.device("xpu:0")
# input_torch = torch.tensor(input_numpy, dtype=torch.float32, device="cuda:0")
input_torch = torch.tensor(input_numpy, dtype=torch.float32)
out_torch = input_torch.mean()
print(out_torch, "torch input mean")

# Convert NumPy array to PaddlePaddle tensor
paddle.device.set_device("xpu")
# input_paddle = paddle.to_tensor(input_numpy, dtype=paddle.float32, place=paddle.XPUPlace(0))
input_paddle = paddle.to_tensor(input_numpy, dtype=paddle.float32)
out_paddle = paddle.mean(input_paddle)
print(out_paddle, "paddle input mean")
'''

<img width="958" alt="Image" src="https://github.com/user-attachments/assets/30d14ffb-2121-4b0b-a985-980dc7d37e46" />
